build/root/Makefile.generated_files

```diff
@@ -32,6 +32,9 @@ endif
 # It's necessary to set this because some environments don't link sh -> bash.
 SHELL := /bin/bash
 
+# TODO(marun) target repo-infra's version of this file
```

Comment:
I hope simplifying the build system is on the roadmap. It hurts to see the complexity moved over.

Reply:
That's definitely on the table.  Maintaining k/k's build system in the context of k/f won't be sustainable.  Deprecating make in favor of bazel is one possibility.